packages/medusa/src/loaders/feature-flags/index.ts

```diff
@@ -0,0 +1,70 @@
+import path from "path"
+import glob from "glob"
+
+import { FlagSettings } from "../../types/feature-flags"
+import { FlagRouter } from "../../utils/flag-router"
+import { Logger } from "../../types/global"
+
+const isTruthy = (val: string | boolean | undefined): boolean => {
+  if (typeof val === "string") {
+    return val.toLowerCase() === "true"
+  }
+  return !!val
+}
+
+export default (
+  configModule: { featureFlags?: Record<string, string | boolean> } = {},
+  logger?: Logger,
+  flagDirectory?: string
+): FlagRouter => {
+  let { featureFlags: projectConfigFlags } = configModule
```

Comment:
suggestion: `let { featureFlags: projectConfigFlags = {} } = configModule` and it assign an empty object as the default values if there is none which means that you can remove the next line then